### ISSUE TITLE
Crash after connecting to board

### ISSUE THREAD

**Issue by tea-lover-418** [lshachar/WiiBalanceWalker#13]
Hi!

Everything works quite well for me, until I click on "Connect to Wii balance board".
What happens is that it DOES connect correctly, as I see the weight values for a second, and I've been able to determine that these are the correct inputs that it receives when it connects (by putting my foot on before starting, and leaving my feet off when starting).

After this initial connection however it crashes. It's unresponsive for a few seconds and then closes.  I'd love to provide more information but I'm not seeing any error codes.

All setting were left to default, and I've tried changing them around a lot to see if something would work. Most importantly (I think) Vjoy output and actions were both turned off. Using "Tare after connection" sets the scale to 0 so that does work for a short amount of time.

I've been able to reproduce this on two PCs with entirely different bluetooth connectors (one laptop and one adapter). I'm on the latest version of Win10 on both. 
If there's anything you can help me with that would be greatly appreciated 🙏 

UPDATE:
It's most definitely not an issue with the connection between my board and the PC as I got valid results from this software: https://www.colorado.edu/neuromechanics/research/wii-balance-board-project.
I also tested the current 32bit and v0.4 versions and both gave the same behaviour.

ANOTHER UPDATE:
I've taken the time to set up a VM with Windows 8.1, and... Everything works! So it seems a recent update from Win10 caused the issues.

**Comment by tea-lover-418** [comment]
Somehow it's working now. One of those cases of "have you tried turning it off then on again", however I'd already done that a couple of times 😅

**Comment by lshachar** [comment]
I'm glad to hear that, because I really had no advice I could offer..
